docs/running-clusterfuzzlite/github_actions.md

```diff
@@ -245,61 +245,115 @@ on:
+        mode: 'prune'
+        # Optional but recommended.
+        # See later section on "Git repo for storage".
+        # storage-repo: https://${{ secrets.PERSONAL_ACCESS_TOKEN }}@github.com/OWNER/STORAGE-REPO-NAME.git
+        # storage-repo-branch: main   # Optional. Defaults to "main"
+        # storage-repo-branch-coverage: gh-pages  # Optional. Defaults to "gh-pages".
 ```
 {% endraw %}
 
-If `storage-repo` is set and `storage-repo-branch-coverage` is "gh-pages" (the
-default), then coverage reports can be viewed at
-`https://USERNAME.github.io/STORAGE-REPO-NAME/coverage/latest/report/linux/report.html`.
+You can copy and paste the above file and pruning should just work.
+However, you can edit the file to:
 
-### Corpus pruning
+- Enable a [storage repo] (`storage-repo`, `storage-repo-branch`,
+  `storage-repo-branch-coverage`).
+
+Finally, let's discuss the last task that ClusterFuzzLite can run: coverage
+report generation.
+
+## Coverage reports
 
-Corpus pruning minimizes a corpus by removing redundant items while keeping the
-same code coverage. To enable this, add the following to `.github/workflows/cflite_cron.yml`:
+To generate periodic coverage reports, add the following to the `jobs` field in
+`.github/workflows/cflite_cron.yml` after the `Pruning` job:
 
 {% raw %}
 ```yaml
 jobs:
-  Pruning:
+  Coverage:
     runs-on: ubuntu-latest
     steps:
     - name: Build Fuzzers
       id: build
       uses: google/clusterfuzzlite/actions/build_fuzzers@v1
+      with:
+        sanitizer: coverage
     - name: Run Fuzzers
       id: run
       uses: google/clusterfuzzlite/actions/run_fuzzers@v1
       with:
         github-token: ${{ secrets.GITHUB_TOKEN }}
         fuzz-seconds: 600
-        mode: 'prune'
-        storage-repo: https://${{ secrets.PERSONAL_ACCESS_TOKEN }}@github.com/OWNER/STORAGE-REPO-NAME.git
-        storage-repo-branch: main   # Optional. Defaults to "main"
-        storage-repo-branch-coverage: gh-pages  # Optional. Defaults to "gh-pages".
+        mode: 'coverage'
+        sanitizer: 'coverage'
+        # Optional but recommended.
+        # See later section on "Git repo for storage".
+        # storage-repo: https://${{ secrets.PERSONAL_ACCESS_TOKEN }}@github.com/OWNER/STORAGE-REPO-NAME.git
+        # storage-repo-branch: main   # Optional. Defaults to "main"
+        # storage-repo-branch-coverage: gh-pages  # Optional. Defaults to "gh-pages".
 ```
 {% endraw %}
 
+You can copy and paste the above file and pruning should just work.
+But here is where `storage-repo`, `storage-repo-branch`, and
+`storage-repo-branch-coverage` come in handy.
+
+If `storage-repo` is set and `storage-repo-branch-coverage` is "gh-pages" (the
+default), then coverage reports can be viewed at
+`https://USERNAME.github.io/STORAGE-REPO-NAME/coverage/latest/report/linux/report.html`.
+This should be much nicer than downloading the coverage report as an artifact.
+
+### Extra configuration
```

Comment:
Should this be "##" and the rest be "###" instead?

Reply:
Right.